lib/lita.rb

```diff
@@ -29,16 +29,25 @@ class << self
     # environment variable LITA_GLOBAL_LOG_LEVEL set to one of the standard log level names.
     attr_accessor :logger
 
-    # Loads user configuration and starts the robot.
+    # Loads user configuration.
     # @param config_path [String] The path to the user configuration file.
-    # @return [void]
-    def run(config_path = nil)
+    # @return [boolean]
```

Comment:
Rather than have this return a boolean, let's have the API just be that if there are no exceptions, it validated.